DigitalLearningSolutions.Web/Helpers/ExternalApis/MapsApiHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers.ExternalApis
+{
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Models.External.Maps;
+    using Newtonsoft.Json;
+
+    public interface IMapsApiHelper
+    {
+        Task<MapsResponse> GetPostcodeCoordinates(string postcode);
+    }
+
+    public class MapsApiHelper : IMapsApiHelper
+    {
+        private readonly HttpClient client;
+        private readonly string mapsApiBaseUrl;
+
+        public MapsApiHelper(HttpClient httpClient)
+        {
+            var mapsApiKey = ConfigHelper.GetAppConfig()["MapsAPIKey"];
+            mapsApiBaseUrl =
+                $"https://maps.googleapis.com/maps/api/geocode/json?key={mapsApiKey}&components=country:GB|postal_code:";
+            httpClient.DefaultRequestHeaders.Accept.Clear();

```

Comment:
I'm not familiar with this - why this needed?

Reply:
It is probably not "necessary" per se, but as the HttpClient can be used for other requests, it just clears the headers on any request to make sure we aren't sending old headers to where they are not necessary. It should probably be performed before actually making each individual call (as it is done in the existing FilteredApiHelper), and I accidentally moved it up here.